src/test/java/tests/BaseTest.java

```diff
@@ -12,10 +15,19 @@
     public WebDriverWait wait;
     public PageGenerator page;
 
-    @BeforeMethod
+    @BeforeClass
     public void setup () {
+    	
+    	
+    	String firefoxdriverpath = System.getProperty("user.dir")
```

Comment:
There is no need to define this. We can use here bonigarcia webdriver manager or the user should save the drivers in their systems. For mac, I am holding them in /usr/bin/local folder.